portfolio/src/main/webapp/css/style.css

```diff
@@ -4,13 +4,30 @@
 }
 
 /* Header Section with navbar and image */
-#banner {
+header {
     height: 100vh;
     min-height: 500px;
-    background-image: url("../img/STEP\ Portfolio\ Design.png");
     background-size: cover;
     background-position: left;
     background-repeat: no-repeat;
+    background-attachment: fixed;
+}
+
+#home-banner {
+    background-image: url("../img/home.png");
```

Comment:
I'd suggesting adding `background-size: cover` here - this helps with sizing it correctly, no matter the viewport size! Same for the other banners you have here :)